src/bpmncwpverify/visitors.py

```diff
@@ -465,3 +450,391 @@ def visit_bpmn(self, bpmn: Bpmn) -> None:
     def end_visit_bpmn(self, bpmn: Bpmn) -> None:
         for place in self.flow_places:
             self.write_places_lines("bit {x} = 0".format(x=str(place)))
+
+
+class CwpGraphVizVisitor(CwpVisitor):  # type: ignore
+    def __init__(self) -> None:
+        self.dot = graphviz.Digraph(comment="cwp graph")
+
+    def visit_state(self, state: CwpState) -> None:
+        self.dot.node(state.name, state.name)
+
+    def end_visit_state(self, state: CwpState) -> None:
+        pass
+
+    def visit_edge(self, edge: CwpEdge) -> None:
+        if edge.source:
+            self.dot.edge(edge.source.name, edge.dest.name, label=edge.expression)
+        else:
+            self.dot.edge("start", edge.dest.name, label=edge.expression)
+
+    def end_visit_edge(self, edge: CwpEdge) -> None:
+        pass
+
+    def visit_cwp(self, model: Cwp) -> None:
+        pass
+
+    def end_visit_cwp(self, model: Cwp) -> None:
+        pass
+
+
+class CwpLtlVisitor(CwpVisitor):  # type: ignore
+    def __init__(self, symbol_table: SymbolTable, print_on: bool = False) -> None:
+        self.state_info: List[str] = []
+        self.symbol_table = symbol_table
+        self.output_str: List[str] = []
+        self.print_on: bool = print_on
+        self.property_list: List[str] = []
+        self.cwp: Cwp
+        self.debug: bool = False
+        self.tab = 0
+
+    def generate_all(self) -> None:
+        self.generate_helper_functions()
+        self.generate_LTL()
+
+    def generate_helper_functions(self) -> None:
+        self.write_line("")
```

Comment:
The white-space is nice for readability, but it makes the code look messy. Is it possible to add a `config` option on the visitor and move the extras space into the prologue of each `write*` function depending on `config`? So, for example, `write_state_variable_ adds two blank lines as part of its definition? That might make things nicer to read.